### DISCUSSION TITLE
Trilium paid hosting

### DISCUSSION THREAD

**Discussion by yongzhengooi** [TriliumNext/Trilium#4821] [Q&A] [answered]
Hey guys, can I get the following if I use the Trilium paid hosting? 
-access it everywhere or share my notes with others as long as I have a domain
-Will the note sync into the domain immediately if I modify the file on my current host?

**Comment by meichthys** [comment]
I believe the answer is yes to all of your questions.
@sigaloid can you confirm?

**Comment by sigaloid** [comment, marked as answer]
1. Yes, you can share notes with the built in share notes feature. You don't need your own domain 
2. It won't sync immediately - it's not fast enough for live collaboration. So you cannot edit the same note simultaneously. Otherwise, yes, the syncing protocol is very robust.

**Reply by yongzhengooi** [reply]
1. So if using a domain can I directly access notes via phone or other laptop without downloading trilium?
2. And should be enough for me if can auto-sync every few min, as it is intended for letting others viewing

**Reply by sigaloid** [reply]
Yes, though features are somewhat limited. For example, no full text search across the whole notebook. There is however math, diagrams, etc. I recommend you try out the app locally and enable share, then open it in your browser to see how it would look for your real notes before you buy. There's also relevant information about the sharing feature in the wiki

**Reply by yongzhengooi** [reply]
I tried the app locally, it was awesome for syncing and viewing in browser mode.

**Comment by Br0kenSilos** [comment]
FWIW, I do not use a service for this. I self-host Trilium on a docker container (in a linux VM on a server in my home). I route Internet access to it using CloudFlare tunnels. I DO NOT share with others. However, even though I have the apps installed, I typically access the whole thing via the webUI for day to day usage. For my day job I remote into a computer at the office from a PC at my home. 

Because I typically run the remote sessions full screen and can only see one at a time, I have my Trilium WebUI opened in a browser in both locations at the same time. I can tell you that in my use case, the changes are near instant. In fact, yesterday I did a side-by-side comparison where we could view my home computer and the office computer at the same time and the worst case was a delay of about 1-2 seconds.

For poops and giggles, I decided to try that with a shared note to see how that would work. It too was "instant". However, the thing to keep in mind with the shared note is that it does not auto-refresh. So I had to make the change on one computer and refresh the browser on the other for the changes to show up.

**Reply by yongzhengooi** [reply]
Yup, you are right. It did sync instantly using locally, seen as worth the money if others can view the note using the domain
